RoadUserCountsHandler saves visualization

youralien opened the pull request:
Summary:

- Add /roadUserCounts route
- Created handler to save road user counts as an icon visualization
- Uses app_config to access the correct project directory
- TODO: Error Handling

Commit:
RoadUserCountsHandler saves visualization

Summary:

- Add /roadUserCounts route
- Created handler to save road user counts as an icon visualization
- Uses app_config to access the correct project directory
- TODO: Use generalized error handler written by pip

runnersaw commented on app/handlers/roadUserCounts.py:
```diff
@@ -17,4 +21,20 @@ class RoadUserCountsHandler(tornado.web.RequestHandler):
     """
     def post(self):
+        identifier = self.get_body_argument('identifier')
+        res = RoadUserCountsHandler.handler(identifier, )
```
What's with the `, )`?

runnersaw replied:
Also, what does `res` become?

runnersaw reviewed (dismissed):
I think just fix the hanging `, )`

youralien commented:
Fixed the hanging `, )`.  Also, added an example of how status codes and error handling COULD work.  Of course, I will replace the `MyAppBaseHandler` with the handler that is created in a separate base handlers file.

jabb1123 reviewed (approved):
🚢 🇮🇹
